packages/popper/src/use-popper.ts

```diff
@@ -13,6 +13,7 @@ import { cssVars, getEventListenerOptions } from "./utils"
 export type { Placement }
 
 export interface UsePopperProps {
+  isEnabled?: boolean
```

Comment:
Why not `isOpen?: boolean` ?
In all the consumers of `usePopper` you map between `isOpen` to `isEnabled` - so isn't  `isOpen` a natural fit?

Reply:
Hi Christian!

I was deciding between the two. The `popper` could be used for positioning things that are not open/close ([popper.js homepage examples](https://popper.js.org/)), so I thought `isEnabled` is more accurate. I don't mind renaming it if you prefer.

Reply:
in my opinion `isOpen` is actually more accurate than `isEnabled` 

an example - you could add `usePopper` within a component so at that point `usePopper` is kind of enabled - but not necessarly open.

If you have no objections I would go with `isOpen` and would merge it afterwards